olp-cpp-sdk-dataservice-read/src/repositories/DataRepository.h

```diff
@@ -41,7 +41,7 @@ class DataRepository final {
       const client::OlpClientSettings& settings);
 
   static DataResponse GetVersionedData(
-      const client::HRN& catalog, const std::string& layer_id,
+      const client::HRN& catalog, const std::string& layer_id, int64_t version,
```

Comment:
```suggestion
      const client::HRN& catalog, const std::string& layer_id, boost::optional<int64_t> version,
```
should be easier to use

Reply:
It's wrong, we should always pass the version